Resources/Private/Language/Module/fr.locallang.xlf

```diff
@@ -86,7 +86,30 @@
 				<source>Last Login</source>
 				<target>Dernière connexion</target>
 			</trans-unit>
-
+			<trans-unit id="members" resname="members">
+				<source>Members</source>
+				<target>Membres</target>
+			</trans-unit>
```

Comment:
les champs username, realName, and mail sont par défaut traduit par BeUser Extension on peut l'utiliser sans le recréer 
\typo3\sysext\beuser\Resources\Private\Language\locallang.xlf